libs/spice/Makefile

```diff
@@ -31,13 +31,12 @@ define Package/libspice-server
   CATEGORY:=Libraries
   TITLE:=SPICE server library
   URL:=https://www.spice-space.org/index.html
-  DEPENDS:=+glib2 +libjpeg +libopenssl +pixman +zlib
+  DEPENDS:=+glib2 +libjpeg +libopenssl +pixman +zlib +libstdcpp
 endef
 
 # audio codec
 MESON_ARGS += \
-	-Dcelt051=disabled \
```

Comment:
Please add a line in the commit message about why the celt051 lines were dropped out in this change.

Reply:
done.